public/js/icinga/storage.js

```diff
@@ -6,6 +6,8 @@
 
     const KEY_TTL = 7776000000; // 90 days (90×24×60×60×1000)
 
+    var backend = window.localStorage;
```

Comment:
Do this in the constructor. This way you'll change the backend for every single storage instance in a window. Not just for the sidebar.

Reply:
Makes sense!